app/controllers/repository_controller.rb

```diff
@@ -0,0 +1,37 @@
+class RepositoryController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  def self.model
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook
```

Comment:
```suggestion
    ManageIQ::Providers::Workflows::AutomationManager::Repository
```

@agrare Is this one ready yet?  I can't recall.

Reply:
I think we will need git support at minimum for MVP.

Reply:
Oh I agree, but do we need to subclass `GitRepository` ?